### PR TITLE
added preserveAspectRatio To props

### PR DESCRIPTION
**Pull request by M-Izadmehr**
In some cases in order to have more customization over the svg tag, one might need to customize the aspect ratio, as a result, I added pereserveAspectRatio to the props.
The new code has backward compatibility and the default value is 'xMidYMid meet', however one can change this value if needed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,3 @@
 .idea
 npm-debug.log
 node_modules
-dist
```
Comment (danilowoz) [inline .gitignore]: Sorry man, but you can't remove this. When the package is released, the `dist` will be published, but on GitHub doesn't need this folder

File: package.json [hunk 2]
```diff
@@ -22,15 +22,12 @@
     "watch":
       "babel --plugins transform-es2015-modules-umd src --watch --ignore tests --out-dir ./dist",
     "build": "babel --plugins transform-es2015-modules-umd src --ignore tests --out-dir ./dist",
-    "lint": "eslint 'src/**/*.js'",
```
Comment (danilowoz) [inline package.json]: The same case here, you can't remove it. This ensures the style and quality code

**Comment by M-Izadmehr** [comment]
Unfortunately, as I was busy with adding this module to my project and I needed dist folder immediately, I forgot to create a new branch (different with the one, that I did pull request on).
As a result in order not to add a useless (revert) push in git history, I re-forked this repository (updated readme) and will submit a new pull request.
Again thank you for this great repository :)

**Comment by danilowoz** [comment]
No problem @M-Izadmehr!
Please submit again your PR, this change is very important!

Thanks :)

**Pull request closed without merging**